messages_en-2.txt

```diff
@@ -572,15 +574,19 @@ sync.fail.bad.network=Couldn't contact server. Please make sure an internet conn
 sync.fail.server.error=The server had an error processing your data. Please try again later. If the problem persists, contact technical support.
 sync.fail.unknown=An unexpected issue occurred during sync. Please try to sync again.
 
-sync.progress.title=Communicating with Server
```

Comment:
this is probably just revealing my ignorance of how this is used, but do we not need these old strings around to support legacy versions?